test/gpu/native/distArray/blockOutsideOnWorkaround.skipif

```diff
@@ -0,0 +1,6 @@
+# with aod, the access to arr's elements are handled with a get. However, it
```

Comment:
Working on this a bit made me a little bit pessimistic about our distributed array story. I don't think this can get fixed without us knowing it. So, I'm ok with `skipif`. We'll need a good amount of effort to get things working here.